### ISSUE TITLE
sometimes audio cues don't play when going to next/previous difference

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#171544]
@meganrogge -- This works so nicely! Thank you so much for this meaningful improvement. BTW, I have found that audio cues does not work from time to time in the following circumstances. Let's say you have multiple lines of changes (line A; line B; line C, for example) and you can move between changes by pressing Alt+F5/Alt+Shift+F5. Sometimes, audio cues does not work on the line B. I experienced this issue when I press Alt+F5 multiple times in one sec interval. However, I cannot exactly narrow down the issue and it is hard for me to provide any reproducible steps. Just noticed that the audio cues are not working occasionally.

_Originally posted by @jooyoungseo in https://github.com/microsoft/vscode/issues/171200#issuecomment-1384209993_

**Comment by jooyoungseo** [comment]
@meganrogge  Wow! The audio cues are very responsive in the diff editor. This is a game changer. Thanks for making the diff more accessible!
